cypress/integration/orders/orders.unreceivePiece.spec.js

```diff
@@ -53,9 +53,11 @@ describe('orders: Unreceive piece from Order', () => {
         // Receive piece
         Helper.selectFromResultsList();
         Receiving.receivePiece(0, caption, barcode);
+        cy.wait(2000);
```

Comment:
Please add the reason in the comments why you add the cy.wait

Reply:
the robot performs the action faster than the information is displayed. By adding this weiter the test passes and thereby confirms that the information has changed.

Reply:
I mean in the comments of the test, in the code

Reply:
done